docs/NetworkPeeringForScalarDLAuditor.md

```diff
@@ -1,4 +1,4 @@
-# Configure ScalarDL Auditor mode for network peering
+# Configure network peering for ScalarDL Auditor mode
```

Comment:
I just updated the document title. I think we have to `peer two networks` to `make ScalarDL Auditor mode work properly`. So, I think this title is better.